qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -7,7 +7,6 @@
 
```

Comment:
While testing on my local vstart environment I received the following error:

```
2019-01-24 11:52:53,250.250 INFO:__main__:test_invalid_user_id (tasks.mgr.dashboard.test_rgw.RgwApiCredentialsTest) ... FAIL
2019-01-24 11:52:53,250.250 INFO:__main__:Running ['./bin/ceph', 'log', 'Ended test tasks.mgr.dashboard.test_rgw.RgwApiCredentialsTest.test_invalid_user_id']
2019-01-24 11:52:54,334.334 INFO:__main__:Stopped test: test_invalid_user_id (tasks.mgr.dashboard.test_rgw.RgwApiCredentialsTest) in 20.058137s
2019-01-24 11:52:54,335.335 INFO:__main__:
2019-01-24 11:52:54,335.335 INFO:__main__:======================================================================
2019-01-24 11:52:54,335.335 INFO:__main__:FAIL: test_invalid_user_id (tasks.mgr.dashboard.test_rgw.RgwApiCredentialsTest)
2019-01-24 11:52:54,335.335 INFO:__main__:----------------------------------------------------------------------
2019-01-24 11:52:54,335.335 INFO:__main__:Traceback (most recent call last):
2019-01-24 11:52:54,336.336 INFO:__main__:  File "/ceph/qa/tasks/mgr/dashboard/test_rgw.py", line 106, in test_invalid_user_id
2019-01-24 11:52:54,336.336 INFO:__main__:    data['message'])
2019-01-24 11:52:54,336.336 INFO:__main__:AssertionError: 'The user "xyz" is unknown to the Object Gateway.' not found in u'No RGW found'
2019-01-24 11:52:54,336.336 INFO:__main__:
2019-01-24 11:52:54,336.336 INFO:__main__:----------------------------------------------------------------------
2019-01-24 11:52:54,336.336 INFO:__main__:Ran 1 test in 74.535s
2019-01-24 11:52:54,336.336 INFO:__main__:
2019-01-24 11:52:54,336.336 INFO:__main__:FAILED (failures=1)
2019-01-24 11:52:54,336.336 INFO:__main__:
2019-01-24 11:52:54,336.336 INFO:__main__:======================================================================
2019-01-24 11:52:54,336.336 INFO:__main__:FAIL: test_invalid_user_id (tasks.mgr.dashboard.test_rgw.RgwApiCredentialsTest)
2019-01-24 11:52:54,337.337 INFO:__main__:----------------------------------------------------------------------
2019-01-24 11:52:54,337.337 INFO:__main__:Traceback (most recent call last):
2019-01-24 11:52:54,337.337 INFO:__main__:  File "/ceph/qa/tasks/mgr/dashboard/test_rgw.py", line 106, in test_invalid_user_id
2019-01-24 11:52:54,337.337 INFO:__main__:    data['message'])
2019-01-24 11:52:54,337.337 INFO:__main__:AssertionError: 'The user "xyz" is unknown to the Object Gateway.' not found in u'No RGW found'
```

I'm not really sure if it's related to the PR or it's a general issue.

Reply:
@s0nea: I think this error is not related to the PR, @alfonsomthd and I ran into the same issue (when running the tests on master) that's why I created a ticket a few days ago: https://tracker.ceph.com/issues/37964
However, @votdev if you have some time it would be great if you could take a look at the issue.